product_state/__manifest__.py

```diff
@@ -7,13 +7,18 @@
     'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
     'website': "https://github.com/OCA/product-attribute",
     'category': 'Product',
-    'version': '12.0.1.0.0',
+    'version': '12.0.2.0.0',
```

Comment:
you should not do that, it will be done by the merger command

Reply:
I thought it is optional, but I will remove it